test/utils/MockQFHelper.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import "contracts/core/libraries/QFHelper.sol";
+
+/// @title Mock QF Helper
+/// @notice A mock contract for testing Quadratic Funding Library
+contract MockQFHelper {
+    using QFHelper for QFHelper.State;
+
+    QFHelper.State internal state;
+
+    function fundRecipients(address[] memory _recipients, uint256[] memory _amounts) public {
+        state.fundRecipients(_recipients, _amounts);
```

Comment:
any chance we could call it just `state.fund(_recipients, _amounts)`? to me it's redundant with `fundRecipients(_recipients, ...`